src/components/AvatarTimeAgo/AvatarTimeAgo.js

```diff
@@ -6,18 +6,16 @@ import TimeAgo from 'react-timeago';
  * Displays time ago and avatar in one label
  */
 const AvatarTimeAgo = ({showUsername, creator, createdAt, size}) => {
-  if (showUsername) {
+  const picUrl =
+    creator && creator.picture
+      ? creator.picture
+      : 'https://www.w3schools.com/css/img_avatar.png';
```

Comment:
Since we use this in a couple locations, maybe it's better to pull it out a configuration variable maybe somewhere under `src/common/`?

Reply:
It might even be better to bundle our own under `src/assets/`.